index.d.ts

```diff
@@ -0,0 +1,83 @@
+// Type definitions for react-native-snackbar 0.3.2
+// Project: https://github.com/cooperka/react-native-snackbar
+// Definitions by: Kyle Roach <https://github.com/iRoachie>
+// TypeScript Version: 2.2.2
+
+
+/**
+ * Actionable button to the snackbar on right.
+ * 
+ * @interface Action
+ */
+interface Action {
+  /**
+   * Text for the button
+   */
+  title: string
+
+  /**
+   * Color of the text for the button
+   * Accepts color strings such as hex, literals, rgba
+   */
+  color: string
+
+  /**
+   * Function called when user presses the button
+   */
+  onPress(): void
+}
+
+
+/**
+ * List of options to make a snackbar
+ * 
+ * @interface SnackBarOptions
+ */
+interface SnackBarOptions {
+   /**
+   * The text that appears on the snackbar.
+   * This property is required
+   */
+  title: string
+
+  /**
+   * Length of time the snackbar stays on screen
+   * It takes either OF three values. Snackbar.LENGTH_SHORT, Snackbar.LENGTH_LONG, Snackbar.LENGTH_INDEFINITE
+   * This property is required
+   */
+  duration: any
+
+  /**
+   * Background color of the snackbar
+   * Accepts color strings such as hex, literals, rgba
+   */
+  backgroundColor?: string
+
+  /**
+   * Adds an actionable button to the snackbar on the right
+   */
+  action?: Action
+}
+
+/**
+ * Snackbar duration of 2000ms (2 seconds)
```

Comment:
These numbers are approximate and seem to vary slightly based on platform, but I think this should say "... of about a second"